x-pack/plugins/observability_ai_assistant/public/components/chat/chat_body.tsx

```diff
@@ -390,6 +391,18 @@ export function ChatBody({
                 return next(messages.concat(message));
               }}
             />
+            {conversation.value?.conversation.token_count && (
+              <EuiFlexGroup justifyContent="flexEnd">
+                <EuiFlexItem grow={false}>
+                  <EuiText size="xs">
+                    {i18n.translate('xpack.observabilityAiAssistant.chatBody.tokenCountTextLabel', {
+                      defaultMessage: '{tokenCount} tokens',
+                      values: { tokenCount: conversation.value?.conversation.token_count },
```

Comment:
This value is not updated when the conversation updates (`ConversationUpdate` event). I looked in `useConversation` and it's not straightforward to update the exposed `conversation`. Any thoughts?

Reply:
> you'll have to add it in the stream event as well though

It's already in the stream with the `ConversationUpdate` event